packages/serverpod/lib/src/server/session.dart

```diff
@@ -112,6 +116,17 @@ abstract class Session {
       _db = server.createDatabase(this);
     }
 
+    var logWriter = Features.enablePersistentLogging
```

Comment:
Discussion: I know that this is just moved code but we should probably support having both enabled. toggling each of them individually.

Reply:
Yeah, but in a future PR I will rewrite the LogWriters, this logic is still there but we can easily change how this works at a single location, it will make sense when you see the next itteration! :)